sourcegraph/cody
# Update Cody Web package

Part of [SRCH-632](https://linear.app/sourcegraph/issue/SRCH-632/merge-cody-web-experimental-package-to-the-cody-repo-main-branch)

This is the one of PRs that come from the bigger change we did for Cody Web in this main PR https://github.com/sourcegraph/cody/pull/4605. 

This PR is based on https://github.com/sourcegraph/cody/pull/4731 (which adds remote context functionality which is used only in the Cody Web package). This should be safe to merge since it only updates cody web package and has no impact on the shared vs code cody extension logic.

This PR updates cody web package
- Making cody web package library package (so later we can use it in the Sourcegraph repository)
- Fixes global styles (include styles only for cody webchat)
- Updates cody web demo (getting it closer to use cases in the Sourcegraph repository)

## Test plan
- CI is passing
- Go to the web package demo `cd web` and run demo `pnpm dev`
- Check that demo works properly with remote context and chat history
